ui/selectors/assets.test.ts

```diff
@@ -484,8 +484,10 @@ describe('getMultichainNativeAssetType', () => {
       selectedMultichainNetworkChainId: SolScope.Mainnet,
       isEvmSelected: false,
       remoteFeatureFlags: {
-        addSolanaAccount: true,
-        addBitcoinAccout: true,
+        addSolanaAccount: true, // NOTE: Will soon be deprecated.
+        solanaAccounts: { enabled: true, minimumVersion: '13.6.0' },
+        addBitcoinAccount: true, // NOTE: Will soon be deprecated.
+        bitcoinAccounts: { enabled: true, minimumVersion: '13.6.0' },
```

Comment:
Since we're removing the feature flag logic, let's cleanup those now actually:

```suggestion
        solanaAccounts: { enabled: true, minimumVersion: '13.6.0' },
        bitcoinAccounts: { enabled: true, minimumVersion: '13.6.0' },
```